notebooks/basic-usage.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
<p>Should always specify whether we talk about PyMC3 or PyMC4 here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>@twiecki I think I have made that distinction clear here?</p>